scripts/irods/test/test_irepl.py

```diff
@@ -2010,7 +2033,30 @@ def test_insufficient_permissions_results_in_no_replication_and_an_error(self):
                             ])
                     self.user0.assert_icommand(['irm', '-f', logical_path])
 
-    def test_sufficient_permissions_results_in_replication_and_no_error(self):
+    def test_insufficient_permissions_for_user_results_in_no_replication_and_an_error(self):
+        user = self.user1.username
+        self.insufficient_permissions_results_in_no_replication_and_an_error_impl(user)
+
+    def test_insufficient_permissions_for_group_results_in_no_replication_and_an_error(self):
+        user = self.user1.username
+        group = 'testgroup'
+
+        try:
+            # Create a test group and add the test user to the group. The test group will have permissions on the data
+            # object while the user itself has none.
```

Comment:
I think this comment is accurate, but it may be potentially misleading if you're checking for insufficient permissions. Maybe something like "The test group will be checked for permissions..." or similar?

Reply:
True, for the case where no permissions are granted to the group or the user. Does this wording make it better?
```suggestion
            # Create a test group and add the test user to the group. The test group will be checked for permissions on
            # the data object while the user itself will have no permissions on the data object.
```

Reply:
i like it

Reply:
I like it too!

Reply:
Done